pyproject.toml

```diff
@@ -13,6 +13,7 @@ license = "Apache 2.0"
 python = ">=3.8,<3.12"
 requests = "^2.25.1"
 singer-sdk = "^0.32.0"
+simplejson = "*"
```

Comment:
`simplejson` is already a dependency of `singer-sdk` so this ensures that _any_ version of it is present in the environment even if the SDK decides to remove it at some point. Also, we're using a public and very basic API so it should be safe to expect any version of the library to work for this package's needs.